Mathlib/Analysis/Asymptotics/Asymptotics.lean

```diff
@@ -1125,6 +1125,9 @@ theorem IsBigO.add_isLittleO (h₁ : f₁ =O[l] g) (h₂ : f₂ =o[l] g) : (fun
   h₁.add h₂.isBigO
 #align asymptotics.is_O.add_is_o Asymptotics.IsBigO.add_isLittleO
 
+theorem IsBigO.isLittleO_add (h₁ : f₁ =O[l] g) (h₂ : f₂ =o[l] g) : (fun x => f₂ x + f₁ x) =O[l] g :=
```

Comment:
We already have `IsLittleO.add_isBigO`, why add this version?

Reply:
removed